model/src/main/proto/exploration.proto

```diff
@@ -247,6 +247,8 @@ message UserAnswer {
     // A list of HTML textual representation of the answer.
     ListOfSetsOfHtmlStrings list_of_html_answers = 4;
   }
+  // A accessible version of the corresponding plain answer. It will be used to read out the text when there is screen-reader enabled.
```

Comment:
```suggestion
  // A accessible version of the corresponding plain answer. It will be used to read out the text when there is a screen-reader enabled.
```

Also, this should probably be broken into 2 lines. It's a bit long (prefer wrapping at 100 characters like in Kotlin/Java).

Reply:
done